app/jobs/importing/run_daily_imports_job.rb

```diff
@@ -17,6 +17,20 @@ def perform
         lock_checks -= 1
       end
       start_time = Time.now
+
+      # expire client consent form if past 1 year
+      release_duration = GrdaWarehouse::Config.get :release_duration
```

Comment:
Let's move this entire logic into `GrdaWarehouse::Hud::Client#revoke_expired_consent`